@salkinium The make build of Mac OS hosted examples is still broken:
```
Building: linux/build_info/main.cpp with Make
Compiling C···· release/modm/ext/printf/printf.o/bin/sh: gcc-10: command not found
make: *** [build/make-release/modm/ext/printf/printf.o] Error 127
```

The make generator has gcc-10 hard-coded for Mac OS hosted, but the pipeline only has 11:
```
%% if platform == "hosted"
        UNAME_S := $(shell uname -s)
        ifeq ($(UNAME_S),Darwin)
                C_SUFFIX := -10
        # Using homebrew include and lib paths on macOS
        CPPDEFINES += -I/usr/local/include
        LIBPATH += -L/usr/local/lib
        endif
%% endif
```

Can I change this to 11 or do you want to fix this properly?

--- a/tools/build_script_generator/make/resources/compiler.mk.in
+++ b/tools/build_script_generator/make/resources/compiler.mk.in
@@ -18,7 +18,7 @@ else
 %% if platform == "hosted"
 	UNAME_S := $(shell uname -s)
 	ifeq ($(UNAME_S),Darwin)
-		C_SUFFIX := -10
+		C_SUFFIX := -11
         # Using homebrew include and lib paths on macOS
         CPPDEFINES += -I/usr/local/include
         LIBPATH += -L/usr/local/lib

